sessionctx/variable/session.go

```diff
@@ -792,8 +797,10 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		s.SlowQueryFile = val
 	case TiDBEnableFastAnalyze:
 		s.EnableFastAnalyze = TiDBOptOn(val)
-	case TiDBWaitTableSplitFinish:
-		s.WaitTableSplitFinish = TiDBOptOn(val)
+	case TiDBWaitSplitRegionFinish:
+		s.WaitSplitRegionFinish = TiDBOptOn(val)
+	case TiDBWaitSplitRegionFinishBackoff:
+		s.Backoff.SetWaitScatterRegionFinishBackoff(int(tidbOptInt64(val, backoff.WaitScatterRegionFinishBackoff)))
```

Comment:
I think we add function like `tidbOptPositiveInt32` is enough.  Then we remove `backoff_time.go`

Reply:
done.